gluten-substrait/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -142,6 +160,8 @@ object ExpressionConverter extends SQLConfHelper with Logging {
     expr match {
       case p: PythonUDF =>
         return replacePythonUDFWithExpressionTransformer(p, attributeSeq, expressionsMap)
+      case s: ScalaUDF if CollapsedExpressionMappings.supported(s.udfName.getOrElse("")) =>
```

Comment:
does any scala udf expression need to be collapsed？

Reply:
not support scala udf now